compatibility_server/service/compatibility-service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: extensions/v1beta1
+kind: Service
+metadata:
+  name: compatibility-server
+  labels:
+    app: compatibility-server
+spec:
+  selector:
+    app: compatibility-server
+    tier: web
+  ports:
+  - port: 80
+    targetPort: 8080
+  type: LoadBalancer
+  loadBalancerIP: 35.184.23.203
```

Comment:
Should this be parameterized some how? Because presumably it only works for one cluster.

Reply:
I think we only have a single service though? And it would be something externally facing:
https://cloud.google.com/kubernetes-engine/docs/tutorials/configuring-domain-name-static-ip#step_3_visit_your_reserved_static_ip_address